o2sim.sh

```diff
@@ -8,6 +8,8 @@ requires:
   - AliGenO2:(?!osx|.*aarch64)
   - EVTGEN:(?!osx)
   - STARlight:(?!osx)
+  - Upcgen:(?!osx)
+  - Graniitti:(?!osx)
```

Comment:
could we add these generators to AliGenO2?

Reply:
> could we add these generators to AliGenO2?

Hi @sawenzel  To my understanding Upcgen and Graniittti are on the same level as STARlight. That is why I added these two generators to o2sim. In any case I think that STARlight, Upcgen, and Graniitti should be at the same place. I however leave it up to you to decide where to put these. Please let me know and I will make the necessary changes.

Reply:
That's a bit historic. We could just have all generators come from AliGenO2. But we can do this cleanup also later. It's only something cosmetic.

Reply:
So I moved now the generators from o2sim.sh to aligeno2.sh. Hope this is ok.